usersync/chooser.go

```diff
@@ -161,3 +170,13 @@ func (c standardChooser) evaluate(bidder string, syncersSeen map[string]struct{}
 
 	return syncer, BidderEvaluation{Bidder: bidder, Status: StatusOK}
 }
+
+func (c standardChooser) requiresCnxSync(bidder string, syncerKey string, cookie *Cookie) bool {
+	if bidder != "connatix" {
```

Comment:
do you need to be worried about case sensitivity here?  in many places in code downcase is used??

Reply:
case sensitivity is not required because `connatix` is the bidder code; anything else would be invalid since you can only ever have 1 bidder code